src/models/syncRule.ts

```diff
@@ -42,6 +48,14 @@ export default class SyncRule {
         return this.syncRule.builder.metadataIds;
     }
 
+    public get useDefaultIncludeExclude(): boolean {
+        return this.syncRule.builder.useDefaultIncludeExclude;
```

Comment:
Note: Existing rules will lead to an error when not finding it. We usually add ``?? true`` for this particular case.

Reply:
ok I'll do it